samples/bluetooth/hci_uart/README.rst

```diff
@@ -6,20 +6,88 @@ Bluetooth: HCI UART
 Overview
 *********
 
-Expose Zephyr Bluetooth Controller support over UART to another device/CPU using
-the H:4 HCI transport protocol (requires HW flow control from the UART)
-
-
+Expose the Zephyr Bluetooth Controller support over UART to another device/CPU
+using the H:4 HCI transport protocol (requires HW flow control from the UART).
 
 Requirements
 ************
 
-* BlueZ running on the host, or
 * A board with BLE support
 
+Default UART settings
+*********************
+
+By default the controller builds use the following settings:
+
+* Baudrate: 1Mbit/s
+* 8 bits, no parity, 1 stop bit
+* Hardware Flow Control (RTS/CTS) enabled
+
 Building and Running
 ********************
+
 This sample can be found under :file:`samples/bluetooth/hci_uart` in the
-Zephyr tree.
+Zephyr tree, and it is built as a standard Zephyr application.
+
+Using the Controller with QEMU and BlueZ
+****************************************
+
+The instructions below show how to use a Nordic nRF52832 as a Zephyr BLE
+Controller and expose it to Linux's BlueZ. This can be very useful in order to
```

Comment:
``Controller`` is sometimes with a capital letter, other times not.  I'd recommend you pick one and stick with it (probably lower case).